ros/noetic/debian/buster/ros-base/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# This is an auto generated Dockerfile for ros:ros-base
+# generated from docker_images/create_ros_image.Dockerfile.em
+FROM ros:noetic-ros-core-buster
+
+# install bootstrap tools
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    build-essential \
+    python-rosdep \
```

Comment:
Ah, thought you meant the template wouldn't compile the same Dockerfiles, as if you had patched them manually.